backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandra.java

```diff
@@ -164,7 +165,7 @@ public void close() throws IOException {
         });
         boolean doNotDeleteImageAfterUsage = false;
         cassandraContainer = new GenericContainer<>(
-            new ImageFromDockerfile(imageName,doNotDeleteImageAfterUsage)
+            new ImageFromDockerfile("james-testing-cassandra", doNotDeleteImageAfterUsage)
```

Comment:
I read our extension code and I don't think this help much, here is the reason why IMO:
- I guess the idea here is to avoid creating duplicated images with different names. But if we utilize the SINGLETON properly, we already avoid this.
- These image/container decleration is just their blueprint declaration. The actual image/container is not created until `container.start()` is called. So our singleton is likely just saving the blueprint declaration overhead. But we still should make sure we utilize that SINGLETON though (I make a commit for LDAP [here](https://github.com/quantranhong1999/james-project/commit/dc2c2c0a9a36a1f3fcc68ba0a4980ebc5231583d). I check other SINGLETON usage and we are good).

Another finding related to this is our testing containers are already being cleaned up by calling `container.stop()`. I run a few tests and see the images are being cleaned up also (not dig where yet but must be somewhere).

Reply:
> I guess the idea here is to avoid creating duplicated images with different names. But if we utilize the SINGLETON properly, we already avoid this.

Not across maven modules. Surefires forks runs on their own JVM.

Reply:
The interesting thing here maybe is how we cleaned up the images and make sure we clean them everywhere.